app/controllers/npq_separation/admin/bulk_operations/revert_applications_to_pending_controller.rb

```diff
@@ -1,44 +1,13 @@
 module NpqSeparation::Admin::BulkOperations
-  class RevertApplicationsToPendingController < NpqSeparation::AdminController
-    before_action :set_bulk_operations, :set_bulk_operation, only: %i[index create]
-    before_action :find_bulk_operation, only: %i[run show]
-
-    def create
-      if (file = params.dig(:bulk_operation_revert_applications_to_pending, :file))
-        BulkOperation::RevertApplicationsToPending.not_started.destroy_all
-        @bulk_operation.file.attach(file)
-        if @bulk_operation.valid?
-          @bulk_operation.save!
-          @bulk_operation.update!(row_count: @bulk_operation.file.download.lines.count)
-          return redirect_to :npq_separation_admin_bulk_operations_revert_applications_to_pending_index
-        end
-      end
-
-      render :index, status: :unprocessable_entity
-    end
-
-    def run
-      @bulk_operation.update!(started_at: Time.zone.now, ran_by_admin_id: current_admin.id)
-      BulkOperation::BulkChangeApplicationsToPendingJob.perform_later(bulk_operation_id: @bulk_operation.id)
-      redirect_to :npq_separation_admin_bulk_operations_revert_applications_to_pending_index
-    end
-
-    def show
-      # empty method, because rubocop will complain in the before_action otherwise
-    end
-
+  class RevertApplicationsToPendingController < NpqSeparation::Admin::BulkOperations::BaseController
```

Comment:
Should be enough to inherit from `BaseController` - you're already in the `BulkOperations` namespace?